src/art/local/backend.py

```diff
@@ -63,6 +63,18 @@ def __init__(self, *, in_process: bool = False, path: str | None = None) -> None
         self._tokenizers: dict[str, "PreTrainedTokenizerBase"] = {}
         self._wandb_runs: dict[str, Run] = {}
 
+    def __enter__(self):
+        return self
+
+    def __exit__(self, *excinfo):
+        self.close()
+    
+    def close(self):
+        for _, service in self._services.items():
+            close_method = getattr(service, "close", None)
+            if callable(close_method):
+                close_method()
+
```

Comment:
It doesn't appear that this context manager or the close method are used anywhere so we can probably remove them, right?

Reply:
Oh never mind, this is the main feature of this PR. Got it 👍